build.py

```diff
@@ -1234,7 +1234,7 @@ def create_dockerfile_linux(
 
 WORKDIR /opt/tritonserver
 COPY --chown=1000:1000 NVIDIA_Deep_Learning_Container_License.pdf .
-
+ENV PIP_BREAK_SYSTEM_PACKAGES=1
```

Comment:
This line should be outdated. 
This change should cover it
https://github.com/triton-inference-server/server/pull/7876